bin/metrics-waf.rb

```diff
@@ -0,0 +1,121 @@
+#! /usr/bin/env ruby
+#
+# metrics-waf
+#
+# DESCRIPTION:
+#   Gets latency metrics from CloudWatch and puts them in Graphite for longer term storage
+#
+# OUTPUT:
+#   metric-data
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: aws-sdk
+#   gem: sensu-plugin
+#   gem: sensu-plugin-aws
+#   gem: time
+#
+# USAGE:
+#
+#
+# NOTES:
+#
+# LICENSE:
+#   Zubov Yuri <yury.zubau@gmail.com> sponsored by Actility, https://www.actility.com
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugin/metric/cli'
+require 'aws-sdk'
+require 'sensu-plugins-aws'
+require 'time'
+
+class WafMetrics < Sensu::Plugin::Metric::CLI::Graphite
```

Comment:
rather than making it directly use graphite only format if possible we should use generic and use an option to control the output format. This means that someone using influxdb does not need to rewrite the check just add the option and choose to specify their format.

https://github.com/sensu-plugins/sensu-plugin/pull/185

In order to use this new functionality we would need to bump `sensu-plugin ~> 2.4` which is fine since we are already using `~> 2.0` it remains backwards compatible.